ballerina/Dependencies.toml

```diff
@@ -116,7 +116,7 @@ modules = [
 [[package]]
 org = "ballerina"
 name = "time"
-version = "2.4.1"
+version = "2.5.1"
```

Comment:
Do we need this change

Reply:
Without this, the PR build fails with the error "cannot resolve module 'ballerina/time'".